clarify what happens after a user is deleted (#189)
mulesoft/docs-access-management: modules/ROOT/pages/users.adoc

The user now has the permissions you selected in the business groups and environments you specified.

> == What happens after a user is deleted

Reviewer: ```suggestion
== Managing Deleted Users
```

> After a user is deleted from the top level Organization, the user is "soft-deleted" and deasocited with the Organization. The reason why it's not "hard-deleted" is because the user identity is referenced in the Audit Log. 

Reviewer: ```suggestion
When you delete a user from your root organization, the user is _soft deleted_, wherein a user can no longer access the root organization or business groups and becomes a dangling Anypoint Platform account unless associated with another organization. Anypoint Platform maintains a record of the user's presence for the purposes of audit logs. 
```

Reviewer 2: suggest "dangling" instead of "orphaned" (more sensitive)

Reviewer 3: I prefer to change "user's former presence in the organization" -> user's former presence. Removing "the organization" because the user is disassociated from the org.

Reviewer: Changed just a bit more @ductran-ms

> For an Anypoint Platform user, if a deleted user tries to login, the login request with fail with error "Your credentials are not valid". Organization administrator can invite the user to join the Organization again, but the user needs to choose a new username. Otherwise the sign up request will fail with error "Cannot create user. User already exists"

Reviewer: ```suggestion
If an Anypoint Platform user that was removed from an organization tries to log in to their former organization, the login request fails, stating `Your credentials are not valid`. 

An organization administrator can invite the user to join the same organization again, but the user must create a new username. If the user tries to log in with their old account, the sign up request fails, stating `Cannot create user. User already exists`.
```

> For a federated Anypoint Platform user, if a deleted user tries to login, the login request will fail with error "This user is disabled", but the user will show up in the Organization again as disabled. Organization administrator can choose to re-enable this user. 

Reviewer: ```suggestion
If a deleted federated user tries to log in, the login request fails, stating `This user is disabled`, but the user still appears in the organization as a disabled account. An organization administrator can re-enable this user. 
```

Reviewer 3: but the user still appears -> but the user will appear

Reviewer: We cannot use future tense.

The user now has the permissions you selected in the business groups and environments you specified.

== Managing Deleted Users

> When you delete a user from your root organization, the user is _soft deleted_, wherein a user can no longer access the root organization or business groups and becomes a dangling Anypoint Platform account unless associated with another organization. Anypoint Platform maintains a record of the user's presence for the purposes of audit logs. 

Reviewer 4: ```suggestion
When you delete a user from your root organization, the user is _soft deleted_. This user can no longer access the root organization or business groups and becomes a dangling Anypoint Platform account unless associated with another organization. Anypoint Platform maintains a record of such users for the purposes of audit logs. 
```

Reviewer 4: I'm really hung up on dangling here because it seems like such a strange word to use, but any edits I came up with just made it longer. Should you do "dangling"?

Reviewer: My proposal was "orphaned," which was rejected.

Reviewer 4: Maybe:

When you delete a user from your root organization, the user is _soft deleted_. This user can no longer access the root organization or business groups but maintains an Anypoint Platform account and can later be associated with another organization. Anypoint Platform maintains a record of such users for the purposes of audit logs.

Reviewer: ```suggestion
When you delete a user from your root organization, the user is _soft deleted_. This user can no longer access the root organization or business groups but maintains an Anypoint Platform account and can later be associated with another organization. Anypoint Platform maintains a record of such users for the purposes of audit logs.
```

> If an Anypoint Platform user that was removed from an organization tries to log in to their former organization, the login request fails, stating `Your credentials are not valid`. 

Reviewer 4: ```suggestion
If an Anypoint Platform user that was removed from an organization tries to log in to their former organization, the login request fails, with a `Your credentials are not valid` error message. 
```

An organization administrator can invite the user to join the same organization again, but the user must create a new username. If the user tries to log in with their old account, the sign up request fails, stating `Cannot create user. User already exists`.

> If a deleted federated user tries to log in, the login request fails, stating `This user is disabled`, but the user still appears in the organization as a disabled account. An organization administrator can re-enable this user. 

Reviewer 4: ```suggestion
If a deleted federated user tries to log in, the login request fails, with a `This user is disabled` error message, but the user still appears in the organization as a disabled account. An organization administrator can re-enable this user. 
```
